config/base_boxes.yaml

```diff
@@ -99,3 +99,11 @@ boxes:
     ansible:
       playbook: 'playbooks/hammer_devel.yml'
       group: 'hammer_devel'
+
+  postgres:
+    box: centos7
+    memory: 1024
+
+  foreman:
+    box: centos7
+    memory: 1024
```

Comment:
Not a big fan of this, and thinking about better ways to handle it. I might switch to using something like [1] to create ones like these on the fly more.

[1] https://github.com/robparrott/ansible-vagrant